<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiplomaWindow</class>
 <widget class="QMainWindow" name="DiplomaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPlainTextEdit" name="sourceEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="errorView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="accessView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="suggestAction"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="setupAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="aboutMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="suggestAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Предложить изменения</string>
   </property>
  </action>
  <action name="setupAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Настроить разбор текста</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>DiplomaWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
